CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Unreleased
-No changes.
+* Support for sending bridged IPMB messages for sensors that are not available on the system interface.
+  `GetSensorReading::new()` and `GetSensorReading::for_sensor()` have been replaced with
+  `GetSensorReading::for_sensor_key()` which now takes a `&Sensorkey`. ([#6])
+  * Currently, this is only implemented for file-based connections - RMCP is not supported yet.
```

Comment:
I think moving the information about limitations to either doc-comments for the `file` module, or the README makes more sense. Restricting the scope of the change (i.e. "Support for (snip) using the `File` connection") should make it clear that not everything is supported.